core/src/main/scala/org/apache/spark/api/r/RUtils.scala

```diff
@@ -62,4 +64,7 @@ private[spark] object RUtils {
       }
     }
   }
+
+  /** Check if R is installed before running tests that use R commands. */
+  def isRInstalled: Boolean = Seq("R", "--version").! == 0
```

Comment:
One minor thing is that this might not work in Windows setups as I think R is shipped as `R.exe` in that case. Any ideas if we actually run tests on Windows as a part of our release process?